app/lib/three_scale/bot_protection/form.rb

```diff
@@ -6,14 +6,14 @@ module Form
       include Base
       include Recaptcha::Adapters::ViewMethods
 
-      delegate :site_account, to: :template
-
-      private
-
       def bot_protection_inputs
         return ''.html_safe unless bot_protection_enabled?
 
-        recaptcha_v3(action: template.controller.controller_path)
+        recaptcha_v3(action: recaptcha_action)
+      end
+
+      def recaptcha_action
+        controller.controller_path
```

Comment:
Just curious - in what situation would this line be called?

Reply:
When the module is not included in any Liquid form, but a `SemanticFormBuilder`, this happens when you open the tenant signup screen (<master-domain>/p/signup)

Reply:
Aaah, OK, that rudimentary signup... (that probably needs to be removed). I forgot about it.
Thanks!